scripts/get_og_previews.py

```diff
@@ -55,29 +60,61 @@ def get_og_preview(url):
     # TODO: Maybe add additional behaviour to check if the image is large enough
     soup = get_page(url)
     image_url = get_og_image(soup)
+
     return image_url
 
 
+def save_image_as_webp(binary_image: bytes, path: Path, file_name: str) -> Path:
+    full_path = path / f"{file_name}.webp"
+    # TODO: Might need to add some validation to make sure the binary is an image
```

Comment:
I think this TODO can be removed. If the image cant be opened/identified, PIL throws an error ([docs](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.open)) which will be handled by the try/except we have in `main()`.